ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -1558,7 +1530,6 @@
 }
 .match-details .variants-data .segmented-bar .segmented-unit-fill {
```

Comment:
Shouldn't all the `.segmented*` rules be removed? The segmented bar widget is supposed to be fully usable without any other style rules except setting the font size and color.